packages/ui/src/accounts/components/UnknownAccountInfo.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import { useAccounts } from '../hooks/useAccounts'
+import { accountOrNamed } from '../model/accountOrNamed'
+
+import { AccountInfo } from './AccountInfo'
+
+interface Props {
+  address: string
```

Comment:
```suggestion
  address: Address
```

For some basic types, we should use better typing. The address has its own defined type.